### PR TITLE
add travis.yml

### PR DESCRIPTION
**Pull request by MaximilianMeister**
and do some basic testing, just gofmt for now

Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
not sure if we need to test all the versions... i just added them for now

**Comment by MaximilianMeister** [comment]
I just figured out that there are also some tests to run, not sure why that was not activated yet? i'll add it as well to the travis script

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+go:
+  - 1.4
+  - 1.5
+  - 1.6
+  - 1.7
+  - 1.8
+  - tip
+
+before_install:
+  - go get github.com/docker/docker/api/types/reference
```
Comment (MaximilianMeister) [inline .travis.yml]: not sure if that is enough actually, as i have not tested it on a fresh machine yet. too bad that travis doesn't run already for that PR

**Comment by flavio** [comment]
I think there's no reason to enable the `docker` service, we are not hitting it inside of the tests.

Also, could you update the `README.md` to include the badge about the test status?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+language: go
+
+go:
+  - 1.4
+  - 1.5
+  - 1.6
+  - 1.7
+  - 1.8
+  - tip
```
Comment (flavio) [inline .travis.yml]: I would just focus on what we will use to build the package on kubic and on tip

**Comment by MaximilianMeister** [comment]
> I think there's no reason to enable the docker service, we are not hitting it inside of the tests.

there is a dependency through `docker_test.go`

```
docker run -it --rm golang /bin/sh
# go get github.com/kubic-project/container-feeder    
# cd $GOPATH/src/github.com/kubic-project/container-feeder
# go test .
--- FAIL: TestDockerDaemonAPIVersion (0.00s)
        docker_test.go:12: unexpected error: exec: "docker": executable file not found in $PATH
        docker_test.go:16: got empty version
FAIL
FAIL    github.com/kubic-project/container-feeder       0.004s
#
```

**Comment by flavio** [comment]
> there is a dependency through docker_test.go

Sorry, I forgot about it

**Pull request merged**